Library/Unit/MonthOnLastWeekDayOfMonthUnit.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace FluentScheduler
+{
+    using System;
+
+    /// <summary>
+    /// Unit of time that represents last day of the month.
+    /// </summary>
+    public sealed class MonthOnLastWeekDayOfMonthUnit
+    {
+        private readonly int _duration;
+
+        internal MonthOnLastWeekDayOfMonthUnit(Schedule schedule, int duration)
+        {
+            _duration = duration;
+            Schedule = schedule;
+            At(0, 0);
+        }
+
+        internal Schedule Schedule { get; private set; }
```

Comment:
Fluent classes must not see Schedule class. All the work is performed by a calculator.